compiler/rustc_parse/locales/en-US.ftl

```diff
@@ -713,6 +713,9 @@ parse_lone_slash = invalid trailing slash in literal
 parse_unskipped_whitespace = non-ASCII whitespace symbol '{$ch}' is not skipped
```

Comment:
```suggestion
parse_unskipped_whitespace = whitespace symbol '{$ch}' is not skipped
```
I don't think the ASCII-ness matters here in any way?
What matters is that it is 1) whitespace and 2) not in the skipped set https://github.com/rust-lang/rust/pull/108403#issuecomment-1459144787.

So I think we can just change the error message here without changing any code.